bitcoin/src/blockdata/block.rs

```diff
@@ -543,7 +543,7 @@ mod tests {
     }
 
     #[test]
-    fn compact_roundrtip_test() {
+    fn compact_roundrtip() {
```

Comment:
In 8fe5ffde4ce290eea827aa41ceda87c7ddf032f8

Can you fix the typo here while you're changing the name?

Reply:
complete